test/alternator/test_service_levels.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+
+import pytest
+from test.alternator.util import random_string, is_aws
+from test.alternator.conftest import new_dynamodb_session
+from test.alternator.test_metrics import metrics, get_metrics, check_increases_metric
+from contextlib import contextmanager
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import Cluster, ExecutionProfile, EXEC_PROFILE_DEFAULT, ConsistencyLevel
+from cassandra.policies import RoundRobinPolicy
+import time
+import re
+
+# Quote an identifier if it needs to be double-quoted in CQL. Quoting is
+# *not* needed if the identifier matches [a-z][a-z0-9_]*, otherwise it does.
+# double-quotes ('"') in the string are doubled.
+def maybe_quote(identifier):
+    if re.match('^[a-z][a-z0-9_]*$', identifier):
+        return identifier
+    return '"' + identifier.replace('"', '""') + '"'
+
+# Convenience context manager for temporarily GRANTing some permission and
+# then revoking it.
+@contextmanager
+def temporary_grant(cql, permission, resource, role):
+    role = maybe_quote(role)
+    cql.execute(f"GRANT {permission} ON {resource} TO {role}")
+    try:
+        yield
+    finally:
+        cql.execute(f"REVOKE {permission} ON {resource} FROM {role}")
+
+# Convenience function for getting the full CQL table name (ksname.cfname)
+# for the given Alternator table. This uses our insider knowledge that
+# table named "x" is stored in keyspace called "alternator_x", and if we
+# ever change this we'll need to change this function too.
+def cql_table_name(tab):
+    return maybe_quote('alternator_' + tab.name) + '.' + maybe_quote(tab.name)
+
+# This file is all about testing RBAC as configured via CQL, so we need to
```

Comment:
All this code is duplicate stuff we have in test_cql_rbac.py, so I added to https://github.com/scylladb/scylladb/issues/22047 the task to clean this duplication - I'm fine that you add this duplicate code for now.

The duplication is doubly silly when the comment says that "this file is all about testing RBAC", which clearly was copied from a file that was really about testing RBAC ;-)